src/deployment/localContainersHelpers.ts

```diff
@@ -238,6 +251,15 @@ export async function validateDockerConnectionProfile(
         }
     }
     state.formErrors = erroredInputs;
+    if (
+        (!propertyName || propertyName === "version") &&
+        arch() === "arm64" &&
+        state.formState.version.includes("2025")
+    ) {
+        state.dialog = {
```

Comment:
Can you create an issue for reverting this once CU1 is released (if one doesn't already exist), stick in the January 2026 milestone, and add a link to it in a comment here?